overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -55021,7 +55027,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "A million millibuckets",
-          "desc:8": "Craft an Extra Utilities Creative Drum! When a Bucket of any fluid is placed inside, it also immediately becomes an infinite source!"
+          "desc:8": "Craft an Extra Utilities Creative Drum! Only usable in creative mode."
```

Comment:
could you emphasize this more? eg "However, since this is only usable in creative mode, you'll have to use the Mekanism Tank for your creative fluid needs." or `§6Note: Not actually usable in survival` (§6 = gold)